diff --git a/script/assemble_ember_build.rb b/script/assemble_ember_build.rb
index 257f619420..df1626a822 100755
--- script/assemble_ember_build.rb
+++ script/assemble_ember_build.rb
@@ -37,16 +37,7 @@
 # Returns a git tree-hash representing the current state of Discourse core JS source paths.
 # Only files in JS_SOURCE_PATHS are included in the tree hash.
 def core_tree_hash
-  Tempfile.create do |f|
-    f.close
-
-    git_dir = capture("git", "rev-parse", "--git-dir").strip
-    FileUtils.cp "#{git_dir}/index", f.path
-
-    env = { "GIT_INDEX_FILE" => f.path }
-
-    # Remove all files from the index, then add only JS_SOURCE_PATHS
-    system(env, "git", "rm", "-r", "--cached", ".", "--quiet", exception: true)
-    system(env, "git", "add", *JS_SOURCE_PATHS, exception: true)
-
-    capture(env, "git", "write-tree").strip
-  end
+  # As YunoHost does not use git, return a fixed string
+  "12345678987654321"
 end
 
+
 def node_heap_size_limit
   capture("node", "-e", "console.log(v8.getHeapStatistics().heap_size_limit/1024/1024)").to_f
 end
