diff --git a/lib/git_utils.rb b/lib/git_utils.rb
index dc926720dd..92f58a22a6 100644
--- a/lib/git_utils.rb
+++ b/lib/git_utils.rb
@@ -11,7 +11,7 @@ class GitUtils
   end
 
   def self.full_version
-    self.try_git('git describe --dirty --match "v[0-9]*" 2> /dev/null', "unknown")
+    self.try_git('git describe --dirty --match "v[0-9]*" 2> /dev/null', Discourse::VERSION::STRING)
   end
 
   def self.has_commit?(hash)
@@ -22,7 +22,8 @@ class GitUtils
 
   def self.last_commit_date
     git_cmd = 'git log -1 --format="%ct"'
-    seconds = self.try_git(git_cmd, nil)
+    # Note(decentral1se): Output from actual command in the v3.2.4 branch
+    seconds = self.try_git(git_cmd, '1721046633')
     seconds.nil? ? nil : DateTime.strptime(seconds, "%s")
   end
 
