diff --git a/app/assets/javascripts/admin/addon/controllers/admin-watched-words-action.js b/app/assets/javascripts/admin/addon/controllers/admin-watched-words-action.js
index 078f8ca989..13ed9b1bb6 100644
--- a/app/assets/javascripts/admin/addon/controllers/admin-watched-words-action.js
+++ b/app/assets/javascripts/admin/addon/controllers/admin-watched-words-action.js
@@ -37,7 +37,7 @@ export default class AdminWatchedWordsActionController extends Controller {
     for (const { regexp, word } of words) {
       try {
         RegExp(regexp);
-      } catch {
+      } catch (err) {
         return i18n("admin.watched_words.invalid_regex", { word });
       }
     }
