diff --git a/app/assets/javascripts/discourse/app/lib/download-calendar.js b/app/assets/javascripts/discourse/app/lib/download-calendar.js
index b2f238b9c0..0a4e1910d2 100644
--- a/app/assets/javascripts/discourse/app/lib/download-calendar.js
+++ b/app/assets/javascripts/discourse/app/lib/download-calendar.js
@@ -23,7 +23,7 @@ export function downloadCalendar(title, dates, options = {}) {
 }
 
 export function downloadIcs(title, dates, options = {}) {
-  const REMOVE_FILE_AFTER = 20_000;
+  const REMOVE_FILE_AFTER = 20000;
   const file = new File([generateIcsData(title, dates, options)], {
     type: "text/plain",
   });
