diff --git a/app/assets/javascripts/admin/addon/controllers/admin-watched-words-action.js b/app/assets/javascripts/admin/addon/controllers/admin-watched-words-action.js
index 2b53a91eb3..eec55647d5 100644
--- a/app/assets/javascripts/admin/addon/controllers/admin-watched-words-action.js
+++ b/app/assets/javascripts/admin/addon/controllers/admin-watched-words-action.js
@@ -32,9 +32,9 @@
     for (const { regexp, word } of words) {
       try {
         RegExp(regexp);
-      } catch {
+      } catch (err) {
         return I18n.t("admin.watched_words.invalid_regex", { word });
       }
     }
   },
 
