diff --git a/lib/discourse.rb b/lib/discourse.rb
index 2b53a91eb3..eec55647d5 100644
--- a/lib/discourse.rb
+++ b/lib/discourse.rb
@@ -668,13 +668,13 @@
   def self.full_version
     @full_version ||= begin
       git_cmd = 'git describe --dirty --match "v[0-9]*" 2> /dev/null'
-      self.try_git(git_cmd, 'unknown')
+      self.try_git(git_cmd, Discourse::VERSION::STRING)
     end
   end
 
   def self.last_commit_date
     @last_commit_date ||= begin
-      git_cmd = 'git log -1 --format="%ct"'
+      seconds = '1563199245' # Note(decentral1se): Output from actual command in the v2.3.2 branch
       seconds = self.try_git(git_cmd, nil)
       seconds.nil? ? nil : DateTime.strptime(seconds, '%s')
     end
