diff --git a/app/assets/javascripts/discourse/app/lib/download-calendar.js b/app/assets/javascripts/discourse/app/lib/download-calendar.js
index 2b53a91eb3..eec55647d5 100644
--- a/app/assets/javascripts/discourse/app/lib/download-calendar.js
+++ b/app/assets/javascripts/discourse/app/lib/download-calendar.js
@@ -22,7 +22,7 @@
 }
 
 export function downloadIcs(title, dates) {
-  const REMOVE_FILE_AFTER = 20_000;
+  const REMOVE_FILE_AFTER = 20000;
   const file = new File([generateIcsData(title, dates)], {
     type: "text/plain",
   });
